strings/base_types.h

```diff
@@ -85,8 +85,11 @@ WINRT_EXPORT namespace winrt
     private:
 
         template <typename TStringView>
-        static constexpr guid parse(TStringView const value)
```

Comment:
Why remove `const` here?

Reply:
Oh, I missed that `value.remove_prefix` was called right under

Reply:
Can't modify a const value.